File: Sakila_Net/Sakila_Net/CS_SakilaLib/Neg/NegCustomer.cs
using System.Data;
using Cl.Harrys.Sakila.DAO;
using Cl.Harrys.Sakila.Model;
namespace Cl.Harrys.Sakila.Neg
{
    public class ClNegCustomer
    {
        public static DataTable listar(int nPagina ,int cantidad)
        {
            return DAO.DAOCustomer.sqlSelectAll(nPagina, cantidad);
        }
        public  static bool consultar( ClCustomer customer) 
        {
            return DAO.DAOCustomer.sqlSelect(customer);
        }
        public  static int crear(ClCustomer customer)
        {
            return DAO.DAOCustomer.sqlInsert(customer);
        }
        public  static int eliminar(ClCustomer customer)
        {
            return DAO.DAOCustomer.sqlDelete(customer);
        }
        public  static int actualizar(ClCustomer customer)
        {
            return DAO.DAOCustomer.sqlUpdate(customer);
        }
    }
}

File: Sakila_Net/Sakila_Net/VB_SakilaLib/Neg/NegCustomer.vb
Imports  SakilaLib.Cl.Harrys.Sakila.DAO
Imports  SakilaLib.Cl.Harrys.Sakila.Model
Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace Neg

                Public Class ClNegCustomer
                    Public Shared Function listar(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAO.DAOCustomer.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public  Shared Function consultar( customer as ClCustomer) as Boolean
                        Return DAO.DAOCustomer.SelectOne(customer)     
                    End Function
                    Public  Shared Function crear( customer as ClCustomer) as Integer
                        Return DAO.DAOCustomer.SqlInsert(customer)     
                    End Function
                    Public  Shared Function eliminar( customer as ClCustomer) as Integer
                        Return DAO.DAOCustomer.SqlDelete(customer)     
                    End Function
                    Public  Shared Function actualizar( customer as ClCustomer) as Integer
                        Return DAO.DAOCustomer.SqlUpdate(customer)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  
